recipes-core/ovmf/ovmf_git.bbappend

```diff
@@ -0,0 +1,78 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/ovmf:"
+
+SRC_URI = "git://github.com/tianocore/edk2.git;branch=master \
+    file://0002-ovmf-update-path-to-native-BaseTools.patch \
+    file://0003-BaseTools-makefile-adjust-to-build-in-under-bitbake.patch \
+    "
+
+SRCREV="dd4cae4d82c7477273f3da455084844db5cca0c0"
+
+GCC_VER="$(${CC} -v 2>&1 | tail -n1 | awk '{print $3}')"
+FILES_${PN} += "\
+    /usr/share/firmware/ovmf.bin \
+    "
+
+fixup_target_tools() {
+    case ${1} in
+      4.4.*)
+        FIXED_GCCVER=GCC44
+        ;;
+      4.5.*)
+        FIXED_GCCVER=GCC45
+        ;;
+      4.6.*)
+        FIXED_GCCVER=GCC46
+        ;;
+      4.7.*)
+        FIXED_GCCVER=GCC47
+        ;;
+      4.8.*)
+        FIXED_GCCVER=GCC48
+        ;;
+      4.9.*)
+        FIXED_GCCVER=GCC49
+        ;;
+      *)
+        FIXED_GCCVER=GCC5
+        ;;
+    esac
+    echo ${FIXED_GCCVER}
+}
+
+do_compile_class-target() {
+    export LFLAGS="${LDFLAGS}"
+    OVMF_ARCH="X64"
```

Comment:
This is a copy and paste of the oe-core version, but hardcoding OVMF_ARCH and dropping some commands at the end after the build.  I want to recommend trying to override TARGET_ARCH, but I can't think of a way.  It's a shame you have to duplicate this.

Reply:
The curse of having multiple target archs in a single build.

Reply:
Maybe the following would not cause too much mayhem and do the trick:
```sh
do_compile_prepend_class-target() {
    TARGET_ARCH=x86_64
```

It does not get rid of the following though
```sh
ln ${build_dir}/${OVMF_ARCH}/EnrollDefaultKeys.efi ${WORKDIR}/ovmf/
```

Reply:
I considered TARGET_ARCH prepend idea, but I think it won't work.  bitbake will expand variables written as ${..} (${TARGET_ARCH}) when it writes the script file run.do_compile, so we can't play games to override the variable at runtime.

Reply:
Getting rid of those symlinks is not important, it was just something we don't need so I got rid of them. If we could just use `do_compile_class-target` as-is by overriding the target arch that would suffice. I will give your suggestion a shot.

Reply:
It keeps building IA32 version even with
```
do_compile_prepend_class-target() {
    export TARGET_ARCH="x86_64"
}